client/src/api/api.js

```diff
@@ -248,12 +249,28 @@ export const uploadAudioFile = async (
 }
 
 /* Audio Endpoints */
+export const downloadAudioFileToExpo = async (vocabId, uri) => {
+  /* Download to a new URI based, with the filename including Date.now() in order to
+    React Native to rerender the audio. If the same filename is kept as before,
+    it won't change the audio displayed on the screen.
+  */
+  const splitPath = uri.split('.')
+  const fileType = splitPath.length > 2 ? splitPath[1] : 'caf'
+
+  const newURI = `${FileSystem.documentDirectory
+    }${vocabId}-audio-${Date.now()}.${fileType}`
+  await FileSystem.copyAsync({ from: uri, to: newURI })
```

Comment:
Just curious, do we need to delete the old audio file(stored at `uri`) after we copy async? I'm assuming the intent here is to create a `newURI` and copy over the existing audio file we have.

Reply:
old audio file is a temporary file that is managed by Expo's Image or Audio library. It will get deleted by those libraries.

Reply:
Oh okie I'll take your word on it 👍